frontend/www/js/omegaup/components/course/Mine.vue

```diff
@@ -31,7 +28,7 @@ import course_FilteredList from './FilteredList.vue';
     'omegaup-course-filtered-list': course_FilteredList,
   },
 })
-export default class CourseList extends Vue {
+export default class MyCoursesList extends Vue {
```

Comment:
```suggestion
export default class Mine extends Vue {
```

por consistencia con el nombre del archivo.